app/src/main/java/com/example/wangyf/android_dokumi_test/MainActivity.java

```diff
@@ -14,5 +19,25 @@ protected void onCreate(Bundle savedInstanceState) {
 
         TextView textView = (TextView)findViewById(R.id.test1);
         textView.setText("aaaaaaaaaaaaaaaaaaaa");
+
+
+        if (true){
+            Integer i = null;
+            i.toString();
+
+        }
+
+        method();
     }
+
+    public void method() {
+        String text = null;
+        System.out.println("text.length=" + text.length());
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wang6793453/android-dokumi-test/blob/ca5b62289af35921bf0a4f1e8ddc1cbe6fbb1376/app/src/main/java/com/example/wangyf/android_dokumi_test/MainActivity.java#L35"></span>Null pointer dereference of text in com.example.wangyf.android_dokumi_test.MainActivity.method()</td>
    </tr>
  </tbody>
</table>